README.md

```diff
@@ -54,7 +54,7 @@ Assuming a clean Windows 10, open a PowerShell with admin rights:
 Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
 
 # Install git
-choco install git python2 -y 
+choco install git python -y 
```

Comment:
should include a winget json for most windows environments, not everyone has choco, for linux and mac with homebrew you can include a brewfile, hate having to use requirements that don't match up in a readme.